src/software/ai/hl/stp/tactic/pass_defender/pass_defender_fsm.cpp

```diff
@@ -78,5 +78,27 @@ void PassDefenderFSM::interceptBall(const Update& event)
             TbotsProto::MaxAllowedSpeedMode::PHYSICAL_LIMIT,
             TbotsProto::DribblerMode::MAX_FORCE, TbotsProto::BallCollisionType::ALLOW,
             AutoChipOrKick{AutoChipOrKickMode::OFF, 0}));
+        return;
     }
+
+    // The ball is likely above the robot
+    // to avoid dividing by 0
+    Angle face_ball_orientation;
+    if ((ball.position() - robot_position).length() == 0)
+    {
+        face_ball_orientation = event.common.robot.orientation();
+    }
+    else
+    {
+        face_ball_orientation = (ball.position() - robot_position).orientation();
+    }
+
+    // backup by the length of the ball
```

Comment:
```suggestion
    // backup by the radius of the ball
```

nit: maybe we could explain why we do this